### PR TITLE
Consolidate affine subspace and multiplicative coset methods into .shift() 

### PR DESCRIPTION
**Pull request by pwang00**
Addresses #18

### CODE REVIEW

File: libiop/algebra/field_subset/field_subset.hpp [hunk 1]
```diff
@@ -64,7 +64,6 @@ class field_subset {
 
     FieldT generator() const;
 
-    const FieldT& offset() const;
     const FieldT shift() const;
```
Comment (ValarDragon) [inline libiop/algebra/field_subset/field_subset.hpp:67]: We should probably open an issue for understanding when to use FieldT vs FieldT&. There isn't a unified approach to this taken throughout the codebase atm.

File: libiop/algebra/field_subset/subspace.hpp [hunk 2]
```diff
@@ -50,15 +50,15 @@ class linear_subspace {
...
     affine_subspace() = default;
-    affine_subspace(const std::vector<FieldT> &basis, const FieldT &offset = FieldT(0));
-    affine_subspace(const linear_subspace<FieldT> &base_space, const FieldT &offset = FieldT(0));
-    affine_subspace(linear_subspace<FieldT> &&base_space, const FieldT &offset = FieldT(0));
+    affine_subspace(const std::vector<FieldT> &basis, const FieldT &shift = FieldT(0));
+    affine_subspace(const linear_subspace<FieldT> &base_space, const FieldT &shift = FieldT(0));
+    affine_subspace(linear_subspace<FieldT> &&base_space, const FieldT &shift = FieldT(0));
 
-    const FieldT& offset() const;
+    const FieldT& shift() const;
```
Comment (ValarDragon) [inline libiop/algebra/field_subset/subspace.hpp:61]: Probably should drop the & here to unify with field_subset.hpp for now

**Review by ValarDragon — CHANGES_REQUESTED** [review]
LGTM! Thanks for doing the refactor.

Just one minor comment about API consistency before merging!

File: libiop/algebra/field_subset/subspace.hpp (hunk 2 above)
Comment (pwang00) [reply libiop/algebra/field_subset/subspace.hpp:61]: Yeah sounds good, will do.

**Pull request merged**